S/SDPA/build_tarballs.jl

```diff
@@ -71,6 +71,11 @@ elif [[ ${target} == *linux* ]]; then
     export LDFLAGS="-ldl -lrt"
 fi
 
+# work around missing strtoll strtoull, see https://github.com/JuliaLang/julia/issues/48081
+if [[ "${target}" == *mingw* ]]; then
+    cp /opt/*-w64-mingw32/*-w64-mingw32/sys-root/lib/libmsvcrt.a /usr/lib/libmsvcrt.a
```

Comment:
Rather than copying the file, add `/opt/*-w64-mingw32/*-w64-mingw32/sys-root/lib` to `LDFLAGS`?  But I'm surprised that directory isn't already in the default search path of the compiler, I'm fairly sure it should be.